install.yaml

```diff
@@ -219,10 +217,10 @@
     notify:
       - Restart systemd-resolved
 
-  - name: Download ubuntu
+  - name: Download Ubuntu
     become: no
     ansible.builtin.get_url:
-      url: https://releases.ubuntu.com/24.04/ubuntu-24.04-live-server-amd64.iso
+      url: https://releases.ubuntu.com/24.04/ubuntu-24.04.1-live-server-amd64.iso
```

Comment:
Maybe this should be extracted into a variable? :thinking:

Reply:
Good idea.